Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/BackgroundTask/AddIshBackgroundTask.cs

```diff
@@ -67,7 +67,7 @@ namespace Trisoft.ISHRemote.Cmdlets.BackgroundTask
     /// <code>
     /// New-IshSession -WsBaseUrl "https://example.com/ISHWS/"
     /// $ishBackgroundTask = Get-IshDocumentObj -LogicalId "GUID-ABCD-1234" |
-    ///                      Add-IshBackgroundTask -EventType "SYNCHRONIZEMETRICS" -InputDataTemplate IshObjectWithIshRef
+    ///                      Add-IshBackgroundTask -EventType "SYNCHRONIZEMETRICS" -InputDataTemplate IshObjectWithLngRef
```

Comment:
For example `Get-IshDocumentObj -LogicalId ISHEDITORTEMPLATEMODULECONCEPT` will return multiple versions and multiple languages.  So the description is wrong mentioned "latest". I did not check outside of this PR, but the typical case for SYNCHRONIZEMETRICS should be -InputDataTemplate IshObjectsWithIshRef where the server expands the LogicalId, right?